srcopsmetrics/evaluate_scores.py

```diff
@@ -52,10 +51,11 @@ def evaluate_reviewers_scores(self, project: str, number_reviewer: int = 2, is_l
         :param number_reviewer: number of reviewers to select
         """
         data = KnowledgeStorage(is_local=is_local).load_previous_knowledge(
-            project_name=project, knowledge_type=EntityTypeEnum.PULL_REQUEST.value,
+            project_name=project, knowledge_type="PullRequest"
```

Comment:
Enums is more strict than using a simple string, it could cause error if typos are there

Reply:
done! changed to use .name()